client/src/admin/FeatureLayer.js

```diff
@@ -0,0 +1,65 @@
+import React, { PropTypes } from 'react';
+import 'leaflet';
+import esri from 'esri-leaflet';
+
+const UNSELECTED_STYLE = {
```

Comment:
Yeah, this is not CSS, it is leaflet code

http://leafletjs.com/reference.html#path-options